src/nft/components/collection/ActivitySwitcher.css.ts

```diff
@@ -3,6 +3,14 @@ import { buttonTextMedium } from 'nft/css/common.css'
 import { loadingAsset } from 'nft/css/loading.css'
 import { sprinkles, vars } from 'nft/css/sprinkles.css'
 
+export const baseActivityContainer = style([
+  {
+    borderBottom: '1px solid',
+    borderColor: vars.color.backgroundInteractive,
+    marginRight: '12px',
+  },
+])
+
```

Comment:
nit: pref `styled-components`

Reply:
Are we shifting to styled components across both codebases?

Reply:
Are we shifting to styled components across both codebases?

Reply:
Updated